### PR TITLE
add description of app

### PR DESCRIPTION
**Pull request by jianructose**

### CODE REVIEW

**Review by jianructose — COMMENTED** [review]
ok to merge

**Review by jianructose — COMMENTED** [review]
close #19

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
...ree graphs on the home page. Using the left-side filtering dropdown and scaler panel, users can filter the conditions they desire by indicating one or multiple selections of country, province, wine variety, as well as the range of the review points and corresponding prices. According to the filtered conditions on the side, the graph on the top as a choropleth map highlights the relative areas as a heatmap, in which the darker-colored area has a higher volume of reviews with tooltip indicating number of reviews, average review points and prices, etc. On the left-bottom side of the page, a bar chart displays the average price over the selected wine variety, which would be a good comparison for users to make their decision based on the selected range of price and desired wine types. The last graph by using a scatter plot, shows the potential relationship between the points and prices given the selected range by an users. This would enable users to better understand if the price pays off its value according to the given points by testers. In the corner side, the two recommendations of a bottle are made by the the highest review points and the best value (price per point), respectively.
```
Comment (nphaterp) [inline README.md]: "shows the potential relationship between points.."

I would just take out 'potential'

Comment (nphaterp) [inline README.md]: "In the corner side,"

I would change to "In the bottom corner"

**Review by nphaterp — DISMISSED** [review]

**Review by mgaroub — APPROVED** [review]

**Review by mgaroub — APPROVED** [review]

**Pull request merged**